modules/ROOT/pages/dataweave-streaming.adoc

```diff
@@ -361,6 +361,9 @@ Reasons:
 
 After processing streamed data, you can stream the output directly to another message processor. To facilitate this behavior, use the `deferred` writer property in the output directive of the DataWeave script, for example, `output application/json deferred=true`.
 
+NOTE:
+Exceptions are not handled when you set `deferred = true`. For example, you can see this behavior in Studio when a flow throws an exception. If you are running an application in Studio debug mode and an exception occurs in a Transform Message component when `deferred = true`, the console logs the error, but the flow does not stop at the Tranform Message component. 
```

Comment:
Should we point to an example.

Reply:
no workaround but you can force the processing to fail with a try statement, though you won't get an exception thrown

Reply:
we'll wait to see if we can/need to provide something on that later. Need more business uses first to see what to provide.

Reply:
added this info to the Jira comments